src/pygeoprocessing/geoprocessing.py

```diff
@@ -1718,8 +1735,9 @@ def reclassify_raster(
         None
 
     Raises:
-        ValueError if ``values_required`` is ``True`` and a pixel value from
-           ``base_raster_path_band`` is not a key in ``attr_dict``.
+        ReclassificationMissingValuesError if ``values_required`` is ``True``
+        and a pixel value from ``base_raster_path_band`` is not a key in
+        ``attr_dict``.
```

Comment:
Sorry, I didn't notice tihs earlier either, but I think ``attr_dict`` should be ``value_map``. I'm guessing that variable changed a while ago and we didn't change the docstring.

Reply:
Good catch.

Reply:
Removed `value_map` here: https://github.com/natcap/pygeoprocessing/pull/106/commits/0ed7d528c4fe110c9333a5877e5d9bc00b902482